iabgpp-encoder/src/main/java/com/iab/gpp/encoder/segment/UsCaV1CoreSegment.java

```diff
@@ -160,7 +160,7 @@ public void validate() {
     } else if (mspaServiceProviderMode == 1) {
       if (mspaOptOutOptionMode != 2) {
         throw new ValidationException("Invalid usca mspa service provider / opt out option modes combination: {"
-            + mspaServiceProviderMode + " / " + mspaServiceProviderMode + "}");
+            + mspaServiceProviderMode + " / " + mspaOptOutOptionMode + "}");
```

Comment:
This is strictly an issue with the exception string. Same fix a few times below.